src/utils.rs

```diff
@@ -253,6 +253,9 @@ pub fn parse_batch(msg:&Batch)-> Ruddermessage{
     for i in &msg.batch {
         match i {
             BatchMessage::Identify(a_) =>{
+                let mut final_context: Value = a_.context.clone().unwrap_or(json!({}));
+                merge(&mut final_context, modified_context.clone() );
```

Comment:
In this merge operation don't the right-hand side values take precedence and override?
If that's the case, if a context parameter is specified at both the event and batch levels, the batch level value takes precedence, which is incorrect.

Reply:
So you are saying that if the same key is present at both the event and batch levels, the event level should have higher precedence?

Reply:
Yes.